morango/utils/sync_utils.py

```diff
@@ -10,6 +11,10 @@
 from morango.models import Buffer, DatabaseMaxCounter, DeletedModels, InstanceIDModel, RecordMaxCounter, RecordMaxCounterBuffer, Store
 from morango.utils.register_models import _profile_models
 
+# custom JSON serialization class
+JSON_SERIALIZER_CLASS_PATH = getattr(settings, "MORANGO_JSON_SERIALIZER_CLASS", "morango.utils.encoders")
+JSON_SERIALIZER_MODULE = importlib.import_module(JSON_SERIALIZER_CLASS_PATH)
+JSON_SERIALIZER = getattr(JSON_SERIALIZER_MODULE, 'MorangoJSONEncoder')
```

Comment:
The typical pattern is to include the class name in the class path, like in test class specifications for running specific test classes. E.g., in cache backend settings: https://docs.djangoproject.com/en/2.0/ref/settings/#caches

Reply:
So the default value would be `morango.utils.encoders.MorangoJSONEncoder`. That way you could have one file with multiple possible encoder classes in it -- the class doesn't need to have a particular special name.

Reply:
Any suggestions on how to do that?
The way I did it, is what I found online that worked for python 2 and python 3.

Reply:
https://github.com/learningequality/morango/issues/39